js/utils/api.js

```diff
@@ -48,6 +48,37 @@ async function getUserProfile(
   }
 }
 
+async function setUserProfile(
+  apiUrlPrefix: string,
+  token: string,
+  newProfile: object
+) {
+  try {
+    let response = await fetch(`${apiUrlPrefix}/api/v1/profile`, {
```

Comment:
the endpoint is wrong, see [specs](https://github.com/teamdigitale/digital-citizenship-functions/blob/master/api/public_api_v1.yaml#L171)

Reply:
That's the proxy endpoint.

Reply:
sorry I got confused, too many APIs to review :D
it would be useful, as reference, to always add a comment with a link to the swagger specs of the API whenever there's some code that calls it